packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.base.tsx

```diff
@@ -106,27 +106,33 @@ export class MessageBarBase extends BaseComponent<IMessageBarProps, IMessageBarS
   }
 
   private _renderMultiLine(): React.ReactElement<React.HTMLAttributes<HTMLAreaElement>> {
+    const { theme } = this.props;
     return (
-      <div className={this._classNames.root}>
-        <div className={this._classNames.content}>
-          {this._getIconSpan()}
-          {this._renderInnerText()}
-          {this._getDismissDiv()}
+      <div style={{ background: theme!.semanticColors.bodyBackground }}>
```

Comment:
would it be better to use a :before on the root instead of introducing an extra element?

Reply:
The `:before` is not working without some sort of `z-index` manipulation which is quite unreliable in some browsers from what I investigated so far. Mainly the issue here is that because the semi-transparent background is set on the root element, inserting a `:before` without `z-index` will cover the background completely. If you give it a `z-index: -1` in trying to put it behind you end up putting it behind the element that in the first place is causing the issue in the screenshot.